src/features/pagination/templates/pagination.html

```diff
@@ -1,17 +1,14 @@
 <div
-  role="contentinfo"
   class="ui-grid-pager-panel"
   ui-grid-pager
   ng-show="grid.options.enablePaginationControls">
   <div
     role="navigation"
     class="ui-grid-pager-container">
     <div
-      role="menubar"
```

Comment:
@JLLeitschuh the description of menubar referenced in the commit message (https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-11) states that a menubar has to be navigable by the arrow keys right and left.
> * Right Arrow:
>   * When focus is in a menubar, moves focus to the next item, optionally wrapping from the last to the first.

This is something, one could fix in the future. However, I think that the role of menubar is not meant to be used here.

1. Items in a menubar are assumed to have submenus, although I am not sure whether this is really compulsory:
> 2. All items in the menubar have child submenus that contain multiple vertically arranged items.

2. A menubar can hold Buttons, Checkboxes or Radiobuttons but no input field:
> * The items contained in a menu are child elements of the containing menu or menubar and have any of the following roles:
>   * menuitem
>   * menuitemcheckbox
>   * menuitemradio

A menuitem again is defined to be a command, which is a button or the like and _no_ input field:
https://www.w3.org/TR/wai-aria-1.1/#command

An example of a menubar can be found here:
https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html